Bad main entry point in package.json

Hi Nick

When using storyshot during unit tests, I got this error message
> Unexpected error while loading ./components/XXX.ts: Error: Cannot find module 'storybook-vue3-router' from 'XXX.ts'

I looked into the `package.json` file and saw that the main entry point is `"main": "dist/cjs/index.js"` but there are no folders `cjs` in the `dist` folder!

Using `"main": "dist/esm/index.js"` solves the issue.

Can you update the package? Thanks :)



_Also, there is a `console.log` left in [withVueRouter](https://github.com/NickMcBurney/storybook-vue3-router/blob/main/src/withVueRouter.ts#L16) ;)_

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "storybook-vue3-router",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "A Storybook decorator that allows you to build stories for your routing-aware components.",
   "keywords": [
     "storybook-addons",

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
   },
   "author": "Nick McBurney",
   "license": "MIT",
-  "main": "dist/cjs/index.js",
+  "main": "dist/esm/index.js",
   "module": "dist/esm/index.js",
   "types": "dist/ts/index.d.ts",
   "files": [

--- a/src/withVueRouter.ts
+++ b/src/withVueRouter.ts
@@ -13,7 +13,6 @@ export const withVueRouter = (routesParam?: Array<RouteRecordRaw>, beforeEach?: 
       const vueRouterGlobalProperty = app.config.globalProperties.$router
       const vueRouteGlobalProperty = app.config.globalProperties.$route
       if (context.viewMode === 'story' && !vueRouterGlobalProperty && !vueRouteGlobalProperty) {
-        console.log(app)
         /* get routes from either `routesParam` or use `defaultRoutes` */
         const routes: Array<RouteRecordRaw> = routesParam || defaultRoutes
 

